gallery/visualtests/nxTreeViewMultiSelect.js

```diff
@@ -15,4 +15,9 @@ describe('NxTreeViewMultiSelect', function() {
   const selector = '#nx-tree-view-multi-select-example .nx-tree-view--select';
 
   it('looks right', simpleTest(selector));
+
+  describe('NxCheckbox', function() {
```

Comment:
Applitools doesn't correctly pick up the full chain of nested `describe` blocks, it only picks up the innermost one.  So this needs to be unambiguous by itself (e.g. 'NxTreeViewMultiSelect checkbox')